scim/resource_user_test.go

```diff
@@ -440,7 +441,7 @@ func TestResourceUserDelete_Error(t *testing.T) {
 	}.ExpectError(t, "I'm a teapot")
 }
 
-func TestResourceUserDelete_NoErrorEmtpyParams(t *testing.T) {
+func TestResourceUserDelete_NoErrorNoDisable(t *testing.T) {
```

Comment:
could you keep existing tests as is and create new ones with introduced invariants? `databricks_user` is one of the most-used resources in provider and this _removes code coverage_ for the standard critical paths.

Reply:
Done